nuxeo-core-binarymanager-s3/src/test/java/org/nuxeo/ecm/core/storage/sql/TestS3BinaryManager2WithSSE.java

```diff
@@ -0,0 +1,44 @@
+package org.nuxeo.ecm.core.storage.sql;
+
+import static org.junit.Assume.assumeTrue;
+import static org.nuxeo.ecm.core.storage.sql.S3BinaryManager.AWS_ID_PROPERTY;
+import static org.nuxeo.ecm.core.storage.sql.S3BinaryManager.AWS_SECRET_PROPERTY;
+import static org.nuxeo.ecm.core.storage.sql.S3BinaryManager.AWS_SESSION_TOKEN_PROPERTY;
+import static org.nuxeo.ecm.core.storage.sql.S3BinaryManager.BUCKET_NAME_PROPERTY;
+import static org.nuxeo.ecm.core.storage.sql.S3BinaryManager.BUCKET_PREFIX_PROPERTY;
+import static org.nuxeo.ecm.core.storage.sql.S3BinaryManager.BUCKET_REGION_PROPERTY;
+import static org.nuxeo.ecm.core.storage.sql.S3BinaryManager.SERVERSIDE_ENCRYPTION_PROPERTY;
+
+import java.util.HashMap;
+
+import org.apache.commons.lang3.StringUtils;
+import org.junit.BeforeClass;
+
+import com.amazonaws.SDKGlobalConfiguration;
+
```

Comment:
missing javadoc. What does `WithSSE` mean?

Reply:
It means Server side encryption, that's something meaningful in the context and prevents to use a 'too long' class name, I can change it to ServerSideEncryption if you believe it's better

Reply:
No need to change the class name, just add a comment in the javadoc